src/Configuration/ValueObject/IncrementalFetchingConfig.php

```diff
@@ -16,7 +16,7 @@ class IncrementalFetchingConfig implements ValueObject
     public static function fromArray(array $data): ?self
     {
         // Enabled ?
-        if (empty($data['incremental'])) {
+        if (empty($data['incrementalFetchingColumn'])) {
```

Comment:
Zmena c. 1: `incremental` zapina incremental loading nie fetching ... takze incremental fetching sa zapina zadaním `incrementalFetchingColumn`. Proste som to zle pochopil.